bundle.Dockerfile

```diff
@@ -6,7 +6,7 @@ LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=curator-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.16.0+git
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.10.0+git
```

Comment:
It looks like you're using an out-of-date operator-sdk version locally compared to what CI's been configured to use. We may need to update how we're getting the tooling dependencies in the future, but can you upgrade your local SDK binary in the meantime?

Reply:
Yes I am doing that.